hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/index/simple/FlinkHoodieSimpleIndex.java

```diff
@@ -135,8 +133,8 @@ public boolean isImplicitWithStorage() {
         context.map(latestBaseFiles, partitionPathBaseFile -> new HoodieKeyLocationFetchHandle<>(config, hoodieTable, partitionPathBaseFile), parallelism);
     Map<HoodieKey, HoodieRecordLocation> recordLocations = new HashMap<>();
     hoodieKeyLocationFetchHandles.stream()
-        .flatMap(handle -> Lists.newArrayList(handle.locations()).stream())
-        .forEach(x -> x.forEach(y -> recordLocations.put(y.getKey(), y.getRight())));
+        .flatMap(handle -> handle.locations())
```

Comment:
seems orthgonal. but agree with the changes

Reply:
This was changed because of the removal of the used guava libs ;)